tests/testing_utils/fixtures.py

```diff
@@ -300,3 +306,36 @@ def _update(snowflake_yml_path: Path, parameter_path: str, value=None):
             yaml.safe_dump(yml, fh)
 
     return _update
+
+
+@pytest.fixture()
+def argument_instance():
+    return Argument(name="Foo", type="Bar")
+
+
+@pytest.fixture()
+def function_instance():
+    return FunctionSchema(
+        name="func1",
+        handler="app.func1_handler",
+        signature=[{"name": "a", "type": "string"}, {"name": "b", "type": "variant"}],
+        returns="string",
+    )
+
+
+@pytest.fixture()
+def native_app_project_instance():
+    return ProjectDefinition(
+        **{
+            "definition_version": "1",
+            "native_app": {
+                "artifacts": [{"dest": "./", "src": "app/*"}],
+                "name": "napp_test",
+                "package": {
+                    "scripts": [
+                        "package/001.sql",
+                    ]
+                },
+            },
+        }
+    )
```

Comment:
These fixtures are used only in `tests/project/test_project_schemas.py` I would move them there.